src/Header.php

```diff
@@ -0,0 +1,738 @@
+     * @link https://tools.ietf.org/html/rfc7231#section-7.1.1.2
+     * @link https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Date
+     */
+    public const DATE = 'Date';
+    /**
+     * Tells the browser that the page being loaded is going to want to perform a large allocation.
+     * @link https://gist.github.com/mystor/5739e222e398efc6c29108be55eb6fe3
+     * @link https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Large-Allocation
+     */
+    public const LARGE_ALLOCATION = 'Large-Allocation';
+    /**
+     * The Link entity-header field provides a means for serialising one or more links in HTTP headers. It is
+     * semantically equivalent to the HTML <link> element.
+     * @link https://tools.ietf.org/html/rfc5988#section-5 The Link Header Field
+     * @link https://tools.ietf.org/html/rfc8288#section-3 Link Serialisation in HTTP Headers
+     * @link https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Link
+     */
+    public const LINK = 'Link';
+    /**
+     * Indicates how long the user agent should wait before making a follow-up request.
+     * @link https://tools.ietf.org/html/rfc7231#section-7.1.3
+     * @link https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Retry-After
+     */
+    public const RETRY_AFTER = 'Retry-After';
+    /**
+     * Communicates one or more metrics and descriptions for the given request-response cycle.
+     * @link https://w3c.github.io/server-timing/#the-server-timing-header-field
+     * @link https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Server-Timing
+     */
+    public const SERVER_TIMING = 'Server-Timing';
+    /**
+     * Used to remove the path restriction by including this header in the response of the Service Worker script.
+     * @link https://www.w3.org/TR/service-workers/#service-worker-allowed
+     */
+    public const SERVICE_WORKER_ALLOWED = 'Service-Worker-Allowed';
+    /**
+     * Links generated code to a source map.
+     * @link https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/SourceMap
+     */
+    public const SOURCEMAP = 'SourceMap';
+    /**
+     * Establishes rules for upgrading or changing to a different protocol on the current client, server, transport
+     * protocol connection.
+     * @link https://tools.ietf.org/html/rfc7230#section-6.7
+     */
+    public const UPGRADE = 'Upgrade';
+    /**
+     * Controls DNS prefetching, a feature by which browsers proactively perform domain name resolution on both links
+     * that the user may choose to follow as well as URLs for items referenced by the document, including images, CSS,
+     * JavaScript, and so forth.
+     * @link
+     * @link https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-DNS-Prefetch-Control
+     */
+    public const X_DNS_PREFETCH_CONTROL = 'X-DNS-Prefetch-Control';
+
+    /**
+     * Pagination
+     */
+
+    /**
+     * The per page limit.
+     */
+    public const X_PAGINATION_LIMIT = 'X-Pagination-Limit';
```

Comment:
I don't sure that these pagination headers should be added.
These not declared in any HTTP standarts. Could you delete?

Reply:
These are OK since these are commonly used in APIs.